include/z64interface.h

```diff
@@ -0,0 +1,82 @@
+#ifndef _Z64INTERFACE_H_
+#define _Z64INTERFACE_H_
+
+/**
+ * Button HUD Colors
+ */
+#define A_BUTTON_R 0   // 90 on N64
+#define A_BUTTON_G 200 // 90 on N64
+#define A_BUTTON_B 50  // 255 on N64
+
+#define B_BUTTON_R 255 // 0 on N64
+#define B_BUTTON_G 30  // 150 on N64
+#define B_BUTTON_B 30  // 0 on N64
```

Comment:
These color values are only ever used to set the corresponding regs which are already named properly, so I don't really see the point in having an extra layer with defines for these.
The other defines in this file seem fine tho, since they are actually used for multiple things.